config/initializers/algoliasearch.rb

```diff
@@ -1,5 +1,5 @@
 AlgoliaSearch.configuration = {
-  application_id: ENV.fetch("ALGOLIA_APP_ID", "Placeholder"),
-  api_key: ENV.fetch("ALGOLIA_WRITE_API_KEY", "Placeholder"),
+  application_id: Rails.env.test? ? "Placeholder" : ENV.fetch("ALGOLIA_APP_ID", "Placeholder"),
```

Comment:
why does this pass `"Placeholder"` as second argument for a none test environment?

Reply:
Because it can't be nil. Although in theory there shouldn't be a non-test environment where it is nil

Reply:
ok, just as joe was passing an empty string in his PR - i guess for the reason you state